openshift-install-powervs

```diff
@@ -1333,6 +1346,21 @@ function process_varfile {
   done
 }
 
+#-------------------------------------------------------------------------
+# Flavor
+#-------------------------------------------------------------------------
+function process_flavor {
+       flavor_file="$flavortype.tfvars"
+        [[ -z "$flavortype" ]] &&  error "The -flavor '$flavourtype' Please specify an option with flavor."
+
+       if [[ $flavortype == "small" || $flavortype == "medium" || $flavortype == "large" ]]; then
```

Comment:
Why hardcode here instead of validating with `../automation/compute-vars/`? ... as you are doing in variables function.